types/ember__modifier/index.d.ts

```diff
@@ -20,3 +20,10 @@ export interface OnModifier extends Opaque<'modifier:on'> {}
  * @see https://api.emberjs.com/ember/4.1/classes/Ember.Templates.helpers/methods/on?anchor=on
  */
 export const on: OnModifier;
+
+export function setModifierManager(
+  factory: (owner: unknown) => unknown,
+  obj: object
+): void;
```

Comment:
Docs plus types to match [the implementation](https://github.com/glimmerjs/glimmer-vm/blob/4f1bef0d9a8a3c3ebd934c5b6e09de4c5f6e4468/packages/%40glimmer/manager/lib/public/index.ts#L23)!

```suggestion
/**
 * Given a modifier manager factory and a modifier, tell Ember to set the
 * manager returned by that factory as the manager for the modifier.
 *
 * @param factory A function which takes an `owner` and returns a [modifier
 *   manager](https://emberjs.github.io/rfcs/0373-Element-Modifier-Managers.html).
 * @param modifier The modifier definition to associate with the manager.
 */
export function setModifierManager<T>(
  factory: (owner: unknown) => unknown,
  modifier: T
): T;
```